flink-table/flink-sql-client/src/main/java/org/apache/flink/table/client/gateway/Executor.java

```diff
@@ -69,74 +70,71 @@
      * <p>Both this method and {@link #getSessionConfigMap(String)} return the same configuration
      * set, but different return type.
      */
-    ReadableConfig getSessionConfig(String sessionId) throws SqlExecutionException;
+    ReadableConfig getSessionConfig(String sessionId) throws SqlClientException;
 
     /**
      * Reset all the properties for the given session identifier.
      *
      * @param sessionId to identifier the session
-     * @throws SqlExecutionException if any error happen.
+     * @throws SqlClientException if any error happen.
      */
-    void resetSessionProperties(String sessionId) throws SqlExecutionException;
+    void resetSessionProperties(String sessionId) throws SqlClientException;
 
     /**
      * Reset given key's the session property for default value, if key is not defined in config
      * file, then remove it.
      *
      * @param sessionId to identifier the session
      * @param key of need to reset the session property
-     * @throws SqlExecutionException if any error happen.
+     * @throws SqlClientException if any error happen.
      */
-    void resetSessionProperty(String sessionId, String key) throws SqlExecutionException;
+    void resetSessionProperty(String sessionId, String key) throws SqlClientException;
 
     /**
      * Set given key's session property to the specific value.
      *
      * @param key of the session property
      * @param value of the session property
-     * @throws SqlExecutionException if any error happen.
+     * @throws SqlClientException if any error happen.
      */
-    void setSessionProperty(String sessionId, String key, String value)
-            throws SqlExecutionException;
+    void setSessionProperty(String sessionId, String key, String value) throws SqlClientException;
 
     /** Parse a SQL statement to {@link Operation}. */
-    Operation parseStatement(String sessionId, String statement) throws SqlExecutionException;
+    Operation parseStatement(String sessionId, String statement) throws SqlClientException;
```

Comment:
nit: I think we can make the exception more specific. Use `SqlClientParseException` is better?